src/components/Button.stories.js

```diff
@@ -1,67 +1,58 @@
+import { ArrowbackButton } from './Button.js';
 import Button from './Button.js';
-import { IconButton } from './Button.js';
+import { DeleteButton } from './Button.js';
+import { EditButton } from './Button.js';
+import { MainNavButton } from './Button.js';
+import { ModalButton } from './Button.js';
 
-import deleteicon from '../images/binicon.svg';
-import editicon from '../images/edit.svg';
-import gobackicon from '../images/goback.svg';
-import newicon from '../images/new.svg';
+import deleteIcon from '../images/binIcon.svg';
+import editIcon from '../images/editIcon.svg';
+import goBackIcon from '../images/goBackIcon.svg';
+import newIcon from '../images/newIcon.svg';
+import saveIcon from '../images/saveIcon.svg';
 
 export default {
   title: 'Component/Button',
   component: Button,
-};
-
-const Template = args => <Button {...args} />;
-
-export const Default = Template.bind({});
-Default.args = {
-  children: 'Add',
-  width: '100%',
-  height: 'fit-content',
-};
-
-export const Delete = Template.bind({ IconButton });
-Delete.args = {
-  children: <img src={deleteicon} alt="delete" />,
-};
-
-export const Yes = Template.bind();
-Yes.args = {
-  children: 'please delete',
-  fontSize: '14px',
-  width: 'fit-content',
-  height: 'fit-content',
-};
-
-export const No = Template.bind();
-No.args = {
-  children: 'no, I want to keep it',
-  fontSize: '14px',
-  width: 'fit-content',
-  height: 'fit-content',
-};
-
-export const Arrow = Template.bind();
-Arrow.args = {
-  children: <img src={gobackicon} alt="delete" />,
-  background: 'transparent',
-  justifySelf: 'start',
-};
-
-export const New = Template.bind();
-New.args = {
-  children: <img src={newicon} alt="new" />,
-  borderRadius: '40%',
-  boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.15)',
-};
-
-export const Edit = Template.bind();
-Edit.args = {
-  children: <img src={editicon} alt="edit" />,
-  background: 'transparent',
-};
-
-export const Save = Template.bind({});
-Save.args = {
-  children: 'save',
-};
+  subcomponents: { ArrowbackButton, MainNavButton, ModalButton },
+};
+
+export const Empty = args => <Button {...args} />;
+
+export const MainNavPlus = args => (
+  <MainNavButton {...args}>
+    <img src={newIcon} alt="new" />
+  </MainNavButton>
+);
+
+export const MainNavSave = args => (
+  <MainNavButton {...args}>
+    <img src={saveIcon} alt="save" />
+  </MainNavButton>
+);
+
+export const ModalDelete = args => (
+  <ModalButton {...args}>please delete</ModalButton>
+);
+
+export const ModalKeep = args => (
+  <ModalButton {...args}>no, I want to keep it</ModalButton>
+);
+
+export const Arrowback = args => (
+  <ArrowbackButton {...args}>
+    <img src={goBackIcon} alt="goback" />
+  </ArrowbackButton>
+);
```

Comment:
This comment i only make as i saw quite at the end of the review files that you wrote goBackIcon - so maybe you like to change also ArrowbackButton to ArrowBackButton - i think both versions can be used